lib/Database.js

```diff
@@ -149,6 +152,20 @@ var Database = module.exports = Class.extend({
   },
 
   /**
+   * Create the order by query segment
+   */
+  getOrderBy: function (orders) {
+    if (typeof orders == 'string' || orders instanceof String) return ' ORDER BY ' + orders;
+    if (!orders || (orders && orders.length === 0)) return '';
+    var query = ' ORDER BY';
+    for (var i in orders) {
+      var order = orders[i];
+      query += ' ' + order[0] + (order[1] ? ' ' + order[1].toUpperCase(): '') + ((orders.length - 1) != i ? ',': '');
```

Comment:
Additionally, there's a mapping between field names and column names. I believe the default is camel case for fields and snake case for columns. Either way, you can look at how the filter column SQL is assembled or SELECT fields and filters, and use something similar for order by fields.